doc/sphinx/language/core/records.rst

```diff
@@ -179,31 +189,54 @@ Accessing fields (projections)
       term_projection ::= @term0 .( @qualid {? @univ_annot } {* @arg } )
       | @term0 .( @ @qualid {? @univ_annot } {* @term1 } )
 
-   The value of a field can be accessed using *projection syntax* (:n:`@term_projection`,
+   The value of a field can be accessed using *projective syntax* (:n:`@term_projection`,
```

Comment:
The word projective rings a bit like projective geometry to me. Would "projection" sound really bad?

Reply:
I used "projection form" and "application form".    Personally, I like an adjective with a noun rather two nouns, but both "projective" and "projection" are OK.

Reply:
OK